app/opinions/getNewestOpinions.js

```diff
@@ -0,0 +1,42 @@
+import express from "express";
+import bodyParser from "body-parser";
+import mongoose from "mongoose";
+
+const app = express();
+
+app.use(
+  bodyParser.urlencoded({
+    extended: true,
+  }),
+);
+app.use(express.static("public"));
+
+mongoose.connect("mongodb://localhost:27017/Opinions-db", {
+  useNewUrlParser: true,
+});
```

Comment:
This logic of creating server and connecting with db should not be in an endpoint file.